Chat App/Chat App/TextInputComponent/TextInputComponentView.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  TextInputComponentViewModel.swift
+//  Chat App
+//
+//  Created by tornike on 29.04.23.
+//
+
+import UIKit
+
+final class TextInputComponentView: UIView {
+    
+    // MARK: Components
+    private let containerView = UIView()
+    private let textView = UITextView()
+    private let button = UIButton()
+    private var containerViewHeightConstraint: NSLayoutConstraint?
+    
+    // MARK: Init
+    init() {
+        super.init(frame: .zero)
+        setUp()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: Setup
+    private func setUp() {
+        setUpContainerView()
+        setUpTextView()
+        setUpButton()
+        setUpLayoutConstraints()
+    }
+    
+    private func setUpContainerView() {
+        containerView.layer.borderWidth = Constants.Button.borderWidth
+        containerView.layer.borderColor = Constants.ContainerView.containerViewBorderColor
+        containerView.layer.cornerRadius = Constants.Button.cornerRadius
+        containerView.translatesAutoresizingMaskIntoConstraints = false
+    }
+    
+    private func setUpTextView() {
+        textView.text = Constants.TextView.placeHolder
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        textView.isScrollEnabled = false
+        textView.isEditable = true
+        textView.textColor = Constants.TextView.placeholderColor
+        textView.delegate = self
+        textView.contentOffset = CGPoint(x: 0, y: (textView.contentSize.height - textView.frame.height) / 2)
+        textView.textContainerInset = UIEdgeInsets(top: 19, left: 22, bottom: 19, right: 0)
+        textView.font = UIFont.systemFont(ofSize: 16)
```

Comment:
we can move this values to Constants

Reply:
you can add `static let textInsets = UIEdgeInsets(top: 19, left: 22, bottom: 19, right: 0)` in Constants